src/Gui/ApplicationPy.cpp

```diff
@@ -1494,20 +1490,18 @@ PyObject* Application::sCreateViewer(PyObject * /*self*/, PyObject *args)
         PyErr_Format(PyExc_ValueError, "views must be > 0");
         return nullptr;
     }
-    else if (num_of_views == 1) {
-        View3DInventor* viewer = new View3DInventor(nullptr, nullptr);
-        if (title)
-            viewer->setWindowTitle(QString::fromUtf8(title));
-        Gui::getMainWindow()->addWindow(viewer);
-        return viewer->getPyObject();
-    }
-    else {
-        SplitView3DInventor* viewer = new SplitView3DInventor(num_of_views, nullptr, nullptr);
+    if (num_of_views == 1) {

```

Comment:
This change is unrelated to using auto, and isn't an automated formatting change.